projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-options.php

```diff
@@ -214,7 +214,6 @@ public function test_sync_default_options() {
 			'site_segment'                                 => 'pineapple',
 			'site_vertical'                                => 'pineapple',
 			'jetpack_excluded_extensions'                  => 'pineapple',
-			'jetpack-memberships-connected-account-id'     => '340',
```

Comment:
This needs to be replaced with `JP_Memerships::$has_connected_account_option_name` value... So it can be synced...